src/App.js

```diff
@@ -23,6 +21,26 @@ import { Footer, Navbar } from "./components/molecules";
 import { ScrollToTop } from "./components/atoms";
 //> Routes
 import Routes from "./Routes";
+//> Core
+import { ferry } from "./actions";
```

Comment:
Could this be given a meaningful name or could be described what it's for?

Reply:
I will rename it to intelFerry due to future ferry implementations.

A description is already added inside `./actions`